plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -203,163 +153,188 @@ export default Component.extend({
       this.details,
       250
     );
-  },
+  }
 
   @bind
   onPresenceChangeCallback(present) {
     if (present) {
       this.chat.updateLastReadMessage();
     }
-  },
+  }
+
+  get capabilities() {
+    return getOwner(this).lookup("capabilities:main");
+  }
 
   @debounce(100)
-  fetchMessages(channel, options = {}) {
+  fetchMessages(options = {}) {
```

Comment:
I guess the Thread component would have its own imlementation of `fetchMessages` and `_fetchMoreMessages` ? Then it would have a similar mechanic for clearMessages, appendMessages, etc on the `channel.activeThread` instead?